### DISCUSSION TITLE
Spinner ideas in lazer

### DISCUSSION THREAD

**Discussion by Stuuugie** [ppy/osu#22983] [Ideas] [answered]
I don't use github so my apologies if this is the wrong place

Spinners always felt like a very different type of object than sliders and circles. While I have seen some very suitable usage for spinners I think there's potentially room for expansion with how spinners could be used.

I think it'd be really cool if spinners could be positioned in more places than dead center. In my head I'm imagining a top right quadrant spinner followed by a bottom left quadrant spinner. This would allow for spinners to be integrated into maps in a more stylized way.

**Comment by Liam-DeVoe** [comment, marked as answer]
https://github.com/ppy/osu/issues/7003
